cyences_app_for_splunk/ReadMe.md

```diff
@@ -46,6 +46,36 @@ To uninstall the app, the users can follow the below steps:
 
 RELEASE NOTES
 -------------
+Version 1.11.0 (January 2022)
+* Kaspersky
+  * Added Kaspersky dashboard.
+    * Added required extractions required.
+  * Added support for Kaspersky in the Device Inventory and Asset Intelligence.
+* DNS Dashboard
+  * Added DNS Tracker dashboard for following use-cases: Top Categories, Record Types, DNS log volume over time, Top queries, Top non-success queries with reason, Top requesters, Top non-success queries from requesters
```

Comment:
We can add note like, it supports all the data sources which is CIM compliance.

Reply:
Done